### PR TITLE
TSE REVIEW

### PR DESCRIPTION
**Pull request by Victorgonbu**
Added all the elements and style for the entire page, applying the bootstrap framework as much as possible.

# Navigation Bars
- Sticky top header, 1310px maximum width (it's not 100% of the viewport because it's only like that when scrolling down, making it 100% width by default I think doesn't look great) 
- Non-sticky Navigation bar added.

# Section and Articles

- Included the elements
- Added the paragraphs
- Embedded images
- Position the section and its elements
- Style the text and background color

majority of this achieved using bootstrap classes.

# Footer
- Styled and positioned the footer lists
- Positioned all the list with its links

# Media query 
- added responsive design for 3 breakpoint (375px, 765px and 992+px )

- Mark up validation made with Linter and W3 Validator

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+  width: 122px;
+}
+
+.submission form button {
+  background-color: #f61602;
+}
+
+.sponsored-section-2 .form button {
+  background-color: #f61602;
+}
```
Comment (andresporras3423) [inline assets/css/main.css]: Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Shaqri** 
Amazing job with your first submission for this project :muscle:  
I can see you make I great effort in your page to make it look like the original :clap: :clap: :clap:  
Your project pass most of the requirements. There are just a couple of details to improve before you can go ahead:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [x]  Check the nav of your page, in medium and smaller screen size the original looks like this:



And your page nav looks like this:



Make sure to hide the user icon in medium and smaller screen size.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

File: assets/css/main.css (hunk 1 above)
Comment (Victorgonbu) [reply assets/css/main.css]: Hi Oscar, thank you very much for reviewing my project. I checked the original page navbar and I saw that the user icon stays until 768px breakpoint, like this: 

Original:

original

Clone: 

clone

I don't really know what to do. Should I make the change? or maybe its a bug of mine?
Please Oscar let me know what to do.

you are right I repeat my self too many times, I dried my code as much as I could, I'll pay more attention to that in my future projects =D

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Hands down, this is one of the best projects I have ever seen. It couldn't be more perfect 🙌👏👌✨!

### This is a project of a world-class developer. Keep up the good work 💪🚀.

### Merge and move ahead.

Happy coding 👨‍💻

**Pull request merged**